web/modules/custom/joinup_seo/config/install/metatag.metatag_defaults.global.yml

```diff
@@ -6,3 +6,9 @@ label: Global
 tags:
   canonical_url: '[current-page:url]'
   title: '[current-page:title] | [site:name]'
+  og_url: '[current-page:url]'
+  og_site_name: '[site:name]'
+  og_image_secure_url: 'https://joinup.ec.europa.eu/themes/joinup/images/logo.svg'
```

Comment:
Wh is this missed from the `config/sync` version? And, again, should not be hardcoded.